workitem/workitem_repository_blackbox_test.go

```diff
@@ -769,3 +769,13 @@ func (s *workItemRepoBlackBoxTest) TestList() {
 
 	})
 }
+
+func (s *workItemRepoBlackBoxTest) TestDeleteWorkitem() {
+	s.T().Run("ok", func(t *testing.T) {
+		fxt := tf.NewTestFixture(t, s.DB,
+			tf.WorkItems(1),
+		)
+		err := s.repo.Delete(s.Ctx, fxt.WorkItems[0].ID, fxt.Identities[0].ID)
+		require.Nil(t, err)
```

Comment:
Is it possible to add more assertion here? Other than just checking error, if it is possible to assert that there is no work item exists, that would be better.

Reply:
addressed here 3d6aabb